utils/epinow2/functions.py

```diff
@@ -113,6 +117,28 @@ def validate_args(
         A dictionary of sanitized arguments.
     """
     args_dict = {}
+    # Split up input string of task_exclusions and validate
+    if task_exclusions is not None:
+        try:
+            task_pairs = task_exclusions.split(",")
+            state_excl = [item.split(":")[0] for item in task_pairs]
+            disease_excl = [item.split(":")[1] for item in task_pairs]
+            for ind_state in state_excl:
+                if ind_state not in all_states:
+                    raise ValueError(f"State {ind_state} not recognized.")
+            for ind_disease in disease_excl:
+                if ind_disease not in all_diseases:
+                    raise ValueError(
+                        f"Disease {ind_disease} not recognized. Valid options are 'COVID-19' or 'Influenza'"
+                    )
+            args_dict["task_exclusions"] = {
+                "geo_value": state_excl,
+                "disease": disease_excl,
+            }
+        except IndexError:
+            print("Task exclusions should be in the form 'state:disease,state:disease'")
+            raise
```

Comment:
```suggestion
        except IndexError:
            raise IndexError("Task exclusions should be in the form 'state:disease,state:disease'")
```

Let's just add the message as a part of the exception, instead of printing.